### DISCUSSION TITLE
Trip-hop duplicated in genre whitelist

### DISCUSSION THREAD

**Discussion by KucharczykL** [beetbox/beets#4503] [Q&A] [answered]
Hello, I've noticed that during tagging genres with the lastgenre plugin, some albums were assigned "Trip Hop" while some others "Trip-Hop". Checking the whitelist file I've found that the trip-hop genre is included twice, once with a hyphen and once with a space. Is this intentional? I haven't checked the whole whitelist but it seems like most genres do not have these duplicates that are intended to catch alternative spellings.

**Comment by jackwilsdon** [comment, marked as answer]
It seems like this probably wasn't intentional.  This script was used to generate the list: https://gist.github.com/sampsyo/1241307

Which pulled from these pages (2011 versions linked, which is when the list was originally generated):

* [Wikipedia: List of popular music genres](http://web.archive.org/web/20110930144201/https://en.wikipedia.org/wiki/List_of_popular_music_genres) (page since deleted, hence the archive.org link)
* [Wikipedia: List of styles of music: S–Z](https://en.wikipedia.org/w/index.php?title=List_of_styles_of_music:_S%E2%80%93Z&oldid=455750366)

There looks to have been a discrepancy between the two pages - one called it "Trip hop" and the other "Trip-hop", which is how we ended up with two versions of it in the list.

Looking at the current Wikipedia page, we should probably just have "Trip hop", as "Trip-hop" just redirects to that article (without the hyphen).

**Reply by sampsyo** [reply]
Yeah, sounds good!

**Comment by KucharczykL** [comment]
Thank you for the answer! I've edited it locally but it'd be great to have it fixed at the source.

**Reply by sampsyo** [reply]
Would you mind putting together a very small PR? You can do it by navigating to the relevant file and editing it in the GitHub browser interface, if that's convenient.

**Reply by KucharczykL** [reply]
Sure, here you go: https://github.com/beetbox/beets/pull/4510

**Reply by sampsyo** [reply]
Awesome; many thanks!!
